mix.exs

```diff
@@ -215,7 +223,19 @@ defmodule Logflare.Mixfile do
       "lint.diff": ["credo diff master"],
       "lint.all": ["credo --strict"],
       "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
-      "ecto.reset": ["ecto.drop", "ecto.setup"]
+      "ecto.reset": ["ecto.drop", "ecto.setup"],
+      "decrypt.dev":
+        "cmd gcloud kms decrypt --ciphertext-file='./.dev.env.enc' --plaintext-file=./.dev.env --location=us-central1 --keyring=logflare-keyring-us-central1 --key=logflare-secrets-key --project=logflare-staging",
```

Comment:
aliases looks good, seems like we missed out prod.
Could you backlog a task for moving these to a Mix.Task? we can probably reduce a lot of repetition.

Reply:
didn't add production yet since we don't have that env set